templates/solochain/runtime/src/lib.rs

```diff
@@ -257,14 +253,15 @@ impl pallet_template::Config for Runtime {
 
 // Create the runtime by composing the FRAME pallets that were previously configured.
 construct_runtime!(
-	pub enum Runtime {
+	pub struct Runtime {
```

Comment:
This was done intentionally to `enum` everywhere as the syntax for instanced pallets doesn't gel with FMT otherwise. It doesn't matter here though as there are no instanced pallets, but probably still better to let it be? This would be solved with v2.

Reply:
How is it that `enum` works but `struct` doesn't?

Reply:
I don't know enough about `cargo fmt` or the config to say, but yeah when you try to instance a pallet with default imports (e.g. `BalancesTwo: pallet_balances::<Instance2> = 1`) then it tries to change it to `pallet_balances<Instance2>` within a `struct` but not within an `enum`, and it doesn't compile (which makes sense).

Also `pallet_balances::<Instance2>::{Config, Call, ..}` works just fine, only when you let it do the default imports.

Reply:
Okay, I thought they are always equivalent.